floss/language/identify.py

```diff
@@ -116,6 +126,31 @@ def get_if_go_and_version(pe: pefile.PE) -> Tuple[bool, str]:
                 pclntab_va = section_data.index(magic) + section_va
                 if verify_pclntab(section, pclntab_va):
                     return True, get_go_version(magic)
+
+    # if not found, the magic bytes may have been patched, search for common Go functions present in all Go samples including obfuscated files
+    # look for the .rdata section first
```

Comment:
should we do this as part of the above loops? or is it better to keep them sequential?

Reply:
I would say I would keep them sequential because it is very likely the version can be extracted in the first loop..

Reply:
ok, sounds good then